api/gen/swagger.yaml

```diff
@@ -49991,271 +50001,66 @@ paths:
                         }
       parameters:
         - name: destination_domain
+          description: The receiving chain identifier according to Circle's CCTP.
           in: path
           required: true
           type: integer
           format: int64
         - name: mint_recipient
+          description: The transfer recipient.
           in: path
           required: true
           type: string
         - name: fallback_recipient
+          description: >-
+            A Noble address used to recover funds in case of errors during the
+            transfer.
           in: path
           required: true
           type: string
         - name: destination_caller
+          description: >-
+            If specified, represents the only address allowed to complete the
+            transfer on the
+
```

Comment:
there is an extra line break here

Reply:
This seems to be added automatically when generating from the Protobuf definition, acknowledging!